core/components/organisms/select/Select.tsx

```diff
@@ -252,22 +262,19 @@ export const Select = React.forwardRef<SelectMethods, SelectProps>((props, ref)
 
   return (
     <SelectContext.Provider value={contextProp}>
-      <div
-        data-test="DesignSystem-Select"
-        aria-haspopup="listbox"
-        aria-expanded={openPopover}
-        style={triggerStyle}
-        {...baseProps}
-      >
+      <div data-test="DesignSystem-Select" aria-haspopup="listbox" aria-expanded={openPopover} {...baseProps}>
         <Popover
           open={openPopover}
           onToggle={onToggleHandler}
           className="mt-3"
           triggerClass="d-block"
+          position="bottom-start"
```

Comment:
Can you please confirm why we are fixing the position of popover to `bottom-start`? Does it handles the case where there is no space on bottom and we want popover to open on top of the trigger?

Reply:
@anuradha9712 I have confirmed with the designer that the position will be `bottom-start`. If there is not enough space at the bottom, the popover will shift to the top of the trigger. I have also fixed this issue.